### PR TITLE
Fix nits in "Loading WebAssembly modules" article

### PR DESCRIPTION
**Pull request by beaufortfrancois**
What's changed, or what was fixed?
- Fix nits

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2018/04/loading-wasm.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ This code snippet does the complete download-compile-instantiate dance, albeit i
   const buffer = await response.arrayBuffer();
   const module = new WebAssembly.Module(buffer);
   const instance = new WebAssembly.Instance(module);
-  const result = instance.exports.fibonacci(42);
+  console.log(instance.exports.fibonacci(42));
```
Comment (mathiasbynens) [inline src/content/en/updates/2018/04/loading-wasm.md]: Too much is happening on this one line, IMHO. I actually prefer this the way it was before. If you want to add a `console.log`, can we just do `console.log(result)` please?

Comment (mathiasbynens) [reply src/content/en/updates/2018/04/loading-wasm.md]: Same for the other instances below

Comment (beaufortfrancois) [reply src/content/en/updates/2018/04/loading-wasm.md]: These changes has been pushed live already. I'll address your feedback in a separate CL.

Comment (mathiasbynens) [reply src/content/en/updates/2018/04/loading-wasm.md]: Please do! Thanks.

Comment (beaufortfrancois) [reply src/content/en/updates/2018/04/loading-wasm.md]: See https://github.com/google/WebFundamentals/pull/6043

**Pull request merged**